application/src/main/java/org/togetherjava/tjbot/commands/moderation/temp/TemporaryModerationRoutine.java

```diff
@@ -138,16 +140,34 @@ private void revokeAction(@NotNull RevocationGroupIdentifier groupIdentifier) {
         return action.revokeAction(guild, target, reason);
     }
 
-    private void handleFailure(@NotNull Throwable failure,
-            @NotNull RevocationGroupIdentifier groupIdentifier) {
-        if (getRevocableActionByType(groupIdentifier.type).handleRevokeFailure(failure,
-                groupIdentifier.targetId) == RevocableModerationAction.FailureIdentification.KNOWN) {
-            return;
-        }
-
-        logger.warn(
+    private void handleFailure(@NotNull Throwable failure, long targetId,
+            @NotNull String actionName) {
+        Runnable logUnknownFailure = () -> logger.warn(
                 "Attempted to revoke a temporary moderation action for user '{}' but something unexpected went wrong.",
-                groupIdentifier.targetId, failure);
+                targetId, failure);
```

Comment:
I personally find this random variable to decrease readability, it's such a small lambda too.
Just put it in the else statement?